lsp/README.md

```diff
@@ -103,6 +103,14 @@ The VS Code extension offers three configuration options:
 
 ### (Neo)Vim
 
+Before proceeding install the [Nickel syntax highlighting plugin](https://github.com/nickel-lang/vim-nickel) using your Vim plugin manager.
+Without this plugin your LSP client will not start NLS.
```

Comment:
```suggestion
Without this plugin your LSP client may not start NLS on nickel source files.
```
I think it can work without on coc.nvim with the right configuration.